docs/airgap-install.md

```diff
@@ -0,0 +1,72 @@
+# Airgap install
+
+In this tutorial we are going to cover k0s deployment in the environment with restricted internet access. 
+As usually in the k0sproject we aim to give you the best user experience with the least possible amount of frictions.
+
+## Prerequisites
+No specific prerequisites are required.
+
+### Prerequisites for exporting images bundle from running cluster
+
+Working cluster with at least one controller (this cluster will be used to build images bundle).
+Please, refer to the [quick start guide](install.md).
```

Comment:
quick start guide->Getting Started
(the name was changed recently)

Reply:
It's not a link, just a part of the sentence (the url is still install.md I guess)

Reply:
Yes, true. However, we use "Getting Started" in the navigation and also in the page title, so IMO it would be better aligned.

Reply:
oh, sorry, didn't understand your original comment. fixed.